frontend/src/components/core/Block/Block.tsx

```diff
@@ -253,7 +260,13 @@ class Block extends PureComponent<Props> {
           style={{ width }}
         />
       ),
-      table: (el: SimpleElement) => <Table element={el} width={width} />,
+      table: (el: SimpleElement) => (
+        <FullScreenWrapper width={width}>
+          {({ width, height }) => (
+            <Table element={el} width={width} height={height} />
```

Comment:
I don't think table is responding to the height property.

Reply:
No, and width either, and I don't fixed it because I finded this comment on the code: https://github.com/streamlit/streamlit/blob/9d049f65dd8ce69353283adb5d4907a124098f6d/frontend/src/components/elements/Table/Table.jsx#L40

@tvst can you clarify this please?

Reply:
Tables don't have a `height` prop. So you should just remove it here.

As for `width`, I don't really recall what's up with that comment I wrote. The comment is now almost 1 year old, though, so you can't blame me! :sunglasses: